app/grandchallenge/algorithms/urls.py

```diff
@@ -133,7 +134,16 @@
         name="model-update",
     ),
     path("<slug>/jobs/", JobsList.as_view(), name="job-list"),
-    path("<slug>/jobs/create/", JobCreate.as_view(), name="job-create"),
+    path(
+        "<slug>/jobs/interface-select/",
+        JobInterfaceSelect.as_view(),
+        name="job-interface-select",
+    ),
+    path(
+        "<slug>/<interface>/jobs/create/",
+        JobCreate.as_view(),
+        name="job-create",
+    ),
```

Comment:
Unsure if we want to expose the interface pks like this? But the only alternative I could come up with was to pass the interface as a URL parameter, but that is less elegant and also still exposes the pk.

Reply:
The `pk` is a uuid so it is fine to expose right?

Reply:
True, it's a uuid. Ok, thanks for confirming!

Reply:
I think the specification that it is a uuid is missing from the url, same with slug, it should be: `"<slug:slug>/<uuid:interface>/jobs/create/"` https://docs.djangoproject.com/en/5.1/topics/http/urls/#path-converters

Reply:
Good point. I'll fix that.